src/seedu/addressbook/commands/Command.java

```diff
@@ -40,6 +40,10 @@ public static String getMessageForPersonListShownSummary(List<? extends ReadOnly
      * Executes the command and returns the result.
      */
     public abstract CommandResult execute();
+    
+    public boolean isMutating() {
```

Comment:
`isMutating()` does not have a header comment.
This method should have a header comment. If not, users of the method as well as those who overrides method will not know how exactly the method behaves.